apps/accounts/tests/test_views.py

```diff
@@ -517,7 +518,7 @@ def test_identify_success_no_account(self):
         response = self.client.get(
             self.url, {'code': 'codes!!', 'state': 'the-right-blob'})
         assert response.status_code == 302
-        assert response['location'] == 'http://testserver/'
+        assert response['location'] == absolutify(reverse('users.edit'))
```

Comment:
I had to remove some mocks to get this to work since it fetches the redirect URL and the user needs to be authenticated for those so it needs to do a full login flow rather than just checking that the login code is called. That means it doesn't test which action was called anymore (login or register), just that the user is authenticated.

It also slows down the 4 tests by about half a second (~4.2 avg before for just these 4 tests and ~4.9 avg with my `assertRedirects()` changes).

It's definitely cleaner but I think it might not be worth it in this case.